### ISSUE TITLE
Can't include typescript folder (index.ts) exports 

### ISSUE THREAD

**Issue by cvle** [pedronauck/docz#71]
## Bug Report

Importing fails silently when trying to include `directory` / `index` exports.
How can I avoid changing my entire codebase to include the `tsx?` extensions?
Why is it currently required?

### This Fails

```ts
// ./BaseButton/index.ts
export * from "./BaseButton";
export { default } from "./BaseButton";
```

```ts
// ./Button/Button.tsx
import { BaseButton } from "../BaseButton";

...
```

```md
# ./Button/Button.mdx

import { Playground } from 'docz'
import Button from './Button.tsx'

# Button

<Playground>
  <Button>Push Me</Button>
</Playground>
```

### This Works

```ts
// ./BaseButton/index.ts
export * from "./BaseButton.tsx";
export { default } from "./BaseButton.tsx";
```

```ts
// ./Button/Button.tsx
import { BaseButton } from "../BaseButton/index.ts";

...
```

```md
# ./Button/Button.mdx

import { Playground } from 'docz'
import Button from './Button.tsx'

# Button

<Playground>
  <Button>Push Me</Button>
</Playground>
```

**Enviroment**

* OS: Debian Squeeze
* Node/npm version:  Node 8.9.4 npm 5.4.0

**Comment by cvle** [comment]
My IDE (vim + YouCompleteMe9 is also complaining with `An import path cannot end with a '.ts' extension. Consider importing '../BaseButton/index' instead.` and the `types` are not imported correctly this way 🤔

**Comment by pedronauck** [comment]
This is a real big problem, I don't know what's happening here but If you omit the extension with typescript webpack don't work as expected!

I don't know if a bug related to [this](https://github.com/pedronauck/docz/blob/master/packages/docz-core/src/utils/babelrc.ts#L20) or [this](https://github.com/pedronauck/docz/blob/master/packages/docz-core/src/bundlers/webpack/config.ts#L138)

**Comment by pedronauck** [comment]
I managed to fix it in [v0.2.8](https://github.com/pedronauck/docz/releases/tag/v0.2.8)
